code/modules/mob/living/carbon/human/species.dm

```diff
@@ -35,7 +35,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/burnmod = 1		// multiplier for burn damage
 	var/coldmod = 1		// multiplier for cold damage
 	var/heatmod = 1		// multiplier for heat damage
-	var/stunmod = 1		// multiplier for stun duration
+	var/stunmod = 1
+	var/clonemod = 1
+	var/toxmod = 1
+	var/staminamod = 1		// multiplier for stun duration
```

Comment:
You don't actually seem to be using these mod vars anywhere except to change their values on IPCs?

Reply:
these vars are already in code. check physiology.DM

Reply:
they literally werent added to species.dm however, which was a think

Reply:
sorry for doubting you ike